src/assets/toolkit/styles/utils/display.css

```diff
@@ -60,4 +68,16 @@
   .u-lg-block {
     display: block !important;
   }
+
+  .u-lg-inlineBlock {
+    display: inline-block !important;
+  }
+
+  .u-lg-unhideVisually {
```

Comment:
Is this only needed in order to override another responsive utility that applies to narrow breakpoints?